libr/main/rasign2.c

```diff
@@ -0,0 +1,118 @@
+/* radare - LGPL - Copyright 2009-2020 - pancake */
+#include <stdio.h>
```

Comment:
we can probably drop all those, just add r_main.h and r_sign.h and maybe r_core.h.. reduce the imports